app/controllers/system/admin/get_help_controller.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+class System::Admin::GetHelpController < System::Admin::Controller
+  def index
+    start_date = params[:start_date].presence
+    end_date = params[:end_date].presence
```

Comment:
Backend needs validation for duration not exceeding 1 year too, to align with FE.  
BE validation is always first.